esx_service/cli/vmdkops_admin.py

```diff
@@ -46,6 +46,8 @@
 def main():
     log_config.configure()
     kv.init()
+    if not vmdk_ops.is_service_available():
+       sys.exit('Unable to connect to the host-agent on this host, ensure the ESX host agent is running before retrying.')
```

Comment:
So do we exit it here, but do not exit in connectLocalSi() ? I think it's OK for the service to  either poll for hostd being up (and wait), or we should exit and have init.d take care of restarts, but why both ?